apps/emqx/src/emqx_schema.erl

```diff
@@ -2316,18 +2316,7 @@ ciphers_schema(Default) ->
         hoconsc:array(string()),
         #{
             default => default_ciphers(Default),
-            converter => fun
-                (undefined) ->
-                    [];
-                (<<>>) ->
-                    [];
-                ("") ->
-                    [];
-                (Ciphers) when is_binary(Ciphers) ->
-                    binary:split(Ciphers, <<",">>, [global]);
-                (Ciphers) when is_list(Ciphers) ->
-                    Ciphers
-            end,
```

Comment:
I replace all fun/1 with fun/2 in converter, because of hocon don't run converter fun when make_serializable is true( check_plain).
https://github.com/emqx/hocon/blob/master/src/hocon_tconf.erl#L1247-L1261
```
eval_arity1_converter(F, V, Opts) ->
    case is_make_serializable(Opts) of
        true ->
            %% the old version convert is only one way
            %% we can only hope V is serializable
            V;
        false ->
            F(V)
    end.
```